pkg/bundle/resources.go

```diff
@@ -37,11 +37,18 @@ func readResources(ctx context.Context, spec *fleet.BundleSpec, compress bool, b
 	var chartDirs []*fleet.HelmOptions
 
 	if spec.Helm != nil && spec.Helm.Chart != "" {
+		if err := parseValueFiles(base, spec.Helm); err != nil {
+			return nil, err
+		}
 		chartDirs = append(chartDirs, spec.Helm)
 	}
 
 	for _, target := range spec.Targets {
-		if target.Helm != nil && target.Helm.Chart != "" {
+		err := parseValueFiles(base, target.Helm)
```

Comment:
In here you still need to check if target.Helm is nil, otherwise it will panic

Reply:
Tweaked the logic.